### ISSUE TITLE
Material icons are confused (missing?) in tip of tree?

### ISSUE THREAD

**Issue by eseidelGoogle** [flutter/flutter#12739]
```

[✓] Flutter (on Mac OS X 10.12.6 16G29, locale en-US, channel master)
    • Flutter at /src/flutter
    • Framework revision 154f2e5402 (3 hours ago), 2017-10-26 09:55:36 -0700
    • Engine revision ec19da1c80
    • Tools Dart version 1.25.0-dev.11.0

[✓] Android toolchain - develop for Android devices (Android SDK 26.0.2)
    • Android SDK at /Users/eseidel/Library/Android/sdk
    • Platform android-26, build-tools 26.0.2
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] iOS toolchain - develop for iOS devices (Xcode 8.3.3)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 8.3.3, Build version 8E3004b
    • ios-deploy 1.9.2
    • CocoaPods version 1.3.1

[✓] Android Studio (version 2.3)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] IntelliJ IDEA Community Edition (version 2017.2.5)
    • Flutter plugin version 17.0
    • Dart plugin version 172.4155.35

[✓] Connected devices
    • iPhone 6 • 80FD76E9-4422-479B-B503-2EFE5C622125 • ios • iOS 10.3 (simulator)
```

Maybe something is wrong with my setup?  But this is what I see when I run a fresh `flutter create` app:
![simulator screen shot oct 26 2017 12 34 48 pm](https://user-images.githubusercontent.com/11857803/32073247-4387379c-ba4a-11e7-8a87-1dd22d28b508.png)

Maybe @xster knows?

**Comment by eseidelGoogle** [comment]
I reset my simulator and it still reproduces.

**Comment by eseidelGoogle** [comment]
I reverted back to 765167bb8caf311f16008bb93db7d6ccf8bdb5e3 (just picked a random point in the past) and the bug went away.  So there is some sort of regression on tip of tree.

**Comment by eseidelGoogle** [comment]
Supposedly this was seen in https://github.com/flutter/flutter/issues/12551, but I'm also seeing this on flutter/flutter@154f2e5402 which I believe is tip of tree?

**Comment by xster** [comment]
Happened between engine commits https://github.com/flutter/engine/commit/8e79156765c67b71b1e1b9895dbc8eea43f9949c and https://github.com/flutter/engine/commit/ec19da1c8098b321672a3770ec0b4e84bd0621e9

**Comment by ferdigokdemir** [comment]
![simulator screen shot - iphone se - 2017-10-26 at 23 29 36](https://user-images.githubusercontent.com/23097465/32075470-974b8fbc-baa5-11e7-8db7-98c48a24ea32.png)

**Comment by xster** [comment]
Reopening this since it's still on head and the chromebots are somehow not building

**Comment by xster** [comment]
Pinged the infra bug. Seems out of our control for now unless we un-roll the engine.

**Comment by eseidelGoogle** [comment]
Infra bots are back on, no?

**Comment by cbracken** [comment]
This was fixed in flutter/engine#4285 and rolled in flutter/flutter#12787.
